packages/dds/tree/src/test/feature-libraries/flex-tree/events.spec.ts

```diff
@@ -632,65 +630,3 @@ describe("beforeChange/afterChange events", () => {
 		assert.strictEqual(childAfterCounter, 1);
 	});
 });
```

Comment:
I'd expect to see a new test for create in an AnchorSet suite somewhere.

Reply:
We're using the forest event now instead. I can't find any unit tests for forest events..

Reply:
Ouch.